content/en/core/releases/4.8.0.md

```diff
@@ -0,0 +1,89 @@
+
+---
+title: "4.8.0 release notes"
+linkTitle: "4.8.0"
+weight:
+description: >
+relevantLinks: >
+toc_hide: true
+---
+
+## Known issues
+
+Check the repository for the [latest known issues](https://github.com/medic/cht-core/issues?q=is%3Aissue+label%3A%22Affects%3A+4.8.0%22).
+
+## Upgrade notes
+
+### Breaking changes
+
+None.
+
+### UI/UX changes
+
+None.
+
+
+## Highlights
+
+### Performance
+
+The Contacts page loads much faster now. Using [apdex scores and testing on low spec devices](https://forum.communityhealthtoolkit.org/t/evaluating-app-performance-using-apdex-lets-dive-in-and-look-at-an-example/3475), we were able to identify slow areas and make improvements to both the [list view](https://forum.communityhealthtoolkit.org/t/performance-improvements-on-the-contacts-page-list-view/3501) and the [detail view](https://forum.communityhealthtoolkit.org/t/performance-improvements-on-the-contacts-page-detail-view/3486). 
```

Comment:
Add a graph showing apdex score before and after using the tables on those forum posts. It's maybe not the most exciting graph but anything visual helps make the release notes more interesting.